jekyll/_cci2/databases.md

```diff
@@ -31,27 +31,25 @@ In the primary image the config defines an environment variable with the `enviro
 Set the POSTGRES_USER environment variable in your CircleCI config to `postgres` to add the role to the image as follows:
 
 ```yml
-      - image: circleci/postgres:9.6-alpine
+      - image: circleci/postgres:14.0
         auth:
           username: mydockerhub-user
           password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
         environment:
           POSTGRES_USER: postgres
 ```
 
-This Postgres image in the example is slightly modified already with `-ram` at the end. It runs in-memory so it does not  hit the disk and that will significantly improve the testing performance on this PostgreSQL database by using this image.
-
 {% raw %}
 
 ```yml
-version: 2
+version: 2.1
 jobs:
```

Comment:
Do we need to include the `orbs` stanza as well where we invoke `circleci/browser-tools`?

Reply:
No, I updated it to remove browsers completely.